lib/xenops_server.ml

```diff
@@ -914,16 +915,18 @@ let rec atomics_of_operation = function
            (VIF_DB.vifs id)
         ) @ simplify (List.map (fun vif -> VIF_plug vif.Vif.id)
                         (VIF_DB.vifs id |> vif_plug_order)
-                     ) @ simplify [
+                     ) @ simplify (List.map (fun vgpu -> VGPU_set_active (vgpu.Vgpu.id, true))
```

Comment:
Let's move this to the left (same level as the `VIF_plug` line), so that it is easier to see what is going on.